docs/shape_batch_size_and_layout.md

```diff
@@ -31,17 +31,33 @@ In case the model can't be reshaped, it will remain in the original parameters a
 will get an error. The model server will also report such problem in the logs.
 
 # Changing model input/output layout
-**DEPRECATED** TODO: Suggest converting model to NHWC model (possible with Model Optimizer in 2022.1).
+OpenVINO™ models can be exported to IR format with arbitrary input/output layout using Model Optimizer. Models which process image data are usually exported with NCHW or NHWC layout. Image transformation libraries like OpenCV or Pillow use NHWC layout. To minimize amount of overhead caused by transposition operation, it is suggested to export your model with NHWC layout.
 
-OpenVINO models which process image data are generated via the model optimizer with NCHW layout. Image transformation libraries like OpenCV or Pillow use NHWC layout. This makes it required to transpose the data in the client application before it can be sent to OVMS. Custom node example implementations internally also use NHWC format to perform image transformations. Transposition operations increase the overall processing latency. Layout parameter reduces the latency by changing the model in runtime to accept NHWC layout instead of NCHW. That way the whole processing cycle is more effective by avoiding unnecessary data transpositions. That is especially beneficial for models with high resolution images, where data transposition could be more expensive in processing.<br><br>
+In case this is not possible, or you already have model with NCHW layout and are not willing to re-export it, OpenVINO™ Model Server allows changing input/output layout at runtime with `--layout` parameter via CLI or `config.json`. Please note that it modifies the model by adding transposition operations as pre-processing step.
 
-Layout parameter is optional. By default layout is inherited from OpenVINO™ model. You can specify layout during conversion to IR format via Model Optimizer. You can also use this parameter for ONNX models.<br>
+Layout parameter is optional. By default layout is inherited from OpenVINO™ model. You can also use this parameter for ONNX models. In case no layout was specified during model export, default layout is `N...` which means that there is only one known dimension which is batch (`N`) and it is on first position.
 
-Layout change is only supported to `NCHW` or `NHWC`. You can specify 2 forms of values:
-  * string - either `NCHW` or `NHWC`; applicable only for models with single input tensor
-  * dictionary of strings - e.g. `{"input1":"NHWC", "input2":"NCHW", "output1":"NHWC"}`; allows to specify layout for multiple inputs and outputs by name.
+Layout change is supported for variety of combinations with `N`, `C`, `H`, `W`, `D`, `?` and letter `...` combinations. Each dimension can appear only once. The exception is `?` which can appear multiple times meaning that there is unknown dimension on given position.
```

Comment:
```suggestion
Layout change is supported for variety of combinations accepting following characters: `N`, `C`, `H`, `W`, `D`, `?`, `...`. Each dimension can appear only once. The exception is `?` which can appear multiple times meaning that there is unknown dimension on given position. `...` means that there is unknown number of dimensions in between.
```

Reply:
But maybe it should be `means that there is unknown number of dimensions at given position.`